src/script/components/avatar/UserAvatar.tsx

```diff
@@ -71,6 +71,7 @@ const UserAvatar: React.FunctionComponent<UserAvatarProps> = ({
       <AvatarInitials avatarSize={avatarSize} initials={participant.initials()} />
       <AvatarImage
         avatarSize={avatarSize}
+        avatarAlt={participant.name()}
```

Comment:
is this just an inconsistency that you use `name()` here and `ko.unwrap()` in the `ServiceAvatar` component? the outcome is the same here and there, am I right?